<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SliceDockWidget</class>
 <widget class="QDockWidget" name="SliceDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>154</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>154</height>
   </size>
  </property>
  <property name="allowedAreas">
   <set>Qt::AllDockWidgetAreas</set>
  </property>
  <property name="windowTitle">
   <string>Slice</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0" columnstretch="0,0,0,0">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>3</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="2">
     <widget class="QSlider" name="horizontalSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QSpinBox" name="spinBox"/>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>200</width>
         <height>90</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="ImageWidget" name="imageWidget" native="true">
          <property name="cursor">
           <cursorShape>CrossCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="toolButtonFill">
      <property name="toolTip">
       <string>Auto size of image</string>
      </property>
      <property name="text">
       <string>Auto size of image</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/icons/icons/images/icons_24.png</normaloff>:/icons/icons/images/icons_24.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QToolButton" name="toolButtonSave">
      <property name="toolTip">
       <string>Export image</string>
      </property>
      <property name="text">
       <string>Export image</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/icons/icons/images/icons_10.png</normaloff>:/icons/icons/images/icons_10.png</iconset>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="4" alignment="Qt::AlignTop">
     <widget class="QWidget" name="labelWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelInfo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="indent">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelLoading">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageWidget</class>
   <extends>QWidget</extends>
   <header location="global">imagewidget.h</header>
   <container>1</container>
   <slots>
    <signal>imageResized(int,int)</signal>
    <signal>clickedPointInImage(int,int)</signal>
    <slot>saveImage()</slot>
    <slot>setAdjust(bool)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonFill</sender>
   <signal>clicked(bool)</signal>
   <receiver>imageWidget</receiver>
   <slot>setAdjust(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonSave</sender>
   <signal>clicked()</signal>
   <receiver>imageWidget</receiver>
   <slot>saveImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>8</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
